packages/shared/web/src/header/invest-and-earn/use-invest-earn-yields.ts

```diff
@@ -5,10 +5,10 @@ import { useObservableState } from 'observable-hooks';
 
 // Returns the highest yields in each of the invest / earn categories
 export function useInvestEarnYields() {
-  const { rates } = useAllMarkets();
+  const { maxRates } = useAllMarkets();
```

Comment:
Not 100% sure but if we don't use the highestLendRate variable returned from useAllMarkets then we should just remove it from the hook. I think it makes more sense to do that logic here rather than inside the hook.